### PR TITLE
Import deployments deprecate network file

### PR DESCRIPTION
**Pull request by coreyar**
## Description

Removes network directory in favor of hardhat deployment exports so all projects follow the same format.

Removes hardcoded addresses in favor of pulling them from packages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +32,8 @@
   "dependencies": {
     "@openzeppelin/contracts": "4.9.3",
     "@openzeppelin/contracts-upgradeable": "^4.8.0",
-    "@venusprotocol/governance-contracts": "^0.0.2",
+    "@venusprotocol/governance-contracts": "^1.3.0",
+    "@venusprotocol/protocol-reserve": "^1.1.0",
```
[the remark this answers is not in the document]
Comment (chechu) [reply package.json:36]: We are not using this package in the contract files, only in the deployment scripts (via the `address.ts` file)

[the remark this answers is not in the document]
Comment (coreyar) [reply package.json:36]: The deployment scripts are packaged.

File: helpers/address.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import OracleTestnet from "@venusprotocol/protocol-reserve/deployments/bsctestnet/ResilientOracle_Proxy.json";
+
+//TESTNET DEPLOYED CONTRACTS
+import Mainnet from "../deployments/bscmainnet.json";
+//MAINNET DEPLOYED CONTRACTS
+import Testnet from "../deployments/bsctestnet.json";
+
+const ADDRESSES = {
+  bsctestnet: {
+    governorProxy: governanceDeployments[97][0].contracts.GovernorBravoDelegator.address,
```
[the remark this answers is not in the document]
Comment (coreyar) [reply helpers/address.ts:14]: This can get cleaned up once this PR is merged https://github.com/VenusProtocol/governance-contracts/pull/28

**Review by GitGuru7 — APPROVED** [review]

File: deployments/bscmainnet/DefaultProxyAdmin.json
Comment (chechu) [inline deployments/bscmainnet/DefaultProxyAdmin.json:1]: This is the ProxyAdmin for VBNBAdmin contract. I would keep it

File: deployments/bscmainnet/XVSVault.json
[the remark this answers is not in the document]
Comment (chechu) [reply deployments/bscmainnet/XVSVault.json:1]: The XVSVault implementation changed with [VIP-201](https://app.venus.io/#/governance/proposal/201)

File: deployments/bscmainnet/DefaultProxyAdmin.json
Comment (coreyar) [reply deployments/bscmainnet/DefaultProxyAdmin.json:1]: Good catch

**Review by chechu — APPROVED** [review]

**Pull request merged**